sardana_alba/macro/tools.py

```diff
@@ -38,7 +38,7 @@ def set_pos(self, moveable, pos):
         moveable_type = moveable.getType()
         if moveable_type == "PseudoMotor":
             moveables_names = moveable.elements
-            values = moveable.calcphysical(pos)
+            values = moveable.CalcPhysical(pos)
```

Comment:
Just FYI, This was not strictly necessary cause this is the Tango Command. But it looks nicer.